### DISCUSSION TITLE
Conflict between notably to Wikidata and notably to NSI

### DISCUSSION THREAD

**Discussion by afanasovich** [osmlab/name-suggestion-index#5879] [Q&A] [answered]
I see a basic conflict between [notably to Wikidata](https://www.wikidata.org/wiki/Wikidata:Notability ) and notably to NSI. 

Some brands do not represent any value from the point of view of Wikipedia - some store has opened, has been working, turned into a network, was opens in almost every city, but nothing remarkable can be written. No scandals, no records.

For example, I faced [Q4034966](https://www.wikidata.org/wiki/Q4034966) was deleted. It was a brand “Beefree” that has 222 clothing stores.

Most commercial brands are not featured on Wikipedia because they do not fulfill the [criteria for notably](https://en.wikipedia.org/wiki/Wikipedia:Notability ). Moreover, networks that operates at the level of a city or several cities. Even if a brand is relevant to Wikipedia, writing articles for all brands is a very difficult task. And there are no links to Wiki projects - there is no reason for existence on Wikidata.

Therefore, most of the brands that I added to Wikidata in the last month can be removed in the same way as Q4034966.

I hope I have conveyed the meaning in my clumsy English.

What to do with deleted records? To protest the deletion? or not add the btand to NSI? 

@bhousel maybe you know?

**Comment by bhousel** [comment]
> For example, I faced [Q4034966](https://www.wikidata.org/wiki/Q4034966) was deleted. It was a brand “Beefree” that has 222 clothing stores.

A brand with 222 stores shouldn't be deleted, and I consider this a mistake.. I think it does pass Wikidata's notability threshold, but sometimes the admins there will delete a page if it doesn't contain enough supporting documentation to look real.

We do have a process for protesting the deletion on their admin noticeboad, and they have been really great about working with us to restore the deleted pages.

I think that name-suggestion-index threshold for "notability" does align with Wikidata's (which is less strict than Wikipedia's).

**Comment by ghost** [comment, marked as answer]
Wikidata admin MisterSynergy obliged by undeleting the befree item. I saw it was duplicated with a later item and merged the two. Once added into NSI and viewable through nsi.guide, the identifier can be added to Wikidata also; this is a good signal to that project that the brand is likely notable (since it appears multiple times in OSM).

**Reply by afanasovich** [reply]
Thx!

I [add  “Beefree”](https://github.com/osmlab/name-suggestion-index/commit/4a5f77ab328d1db5b1ee6a5dc56273c75cf71243)
